test/helper.js

```diff
@@ -18,27 +17,28 @@ export async function resetGitRepository() {
 async function initRepo() {
   for (const repoPath of [ VERSIONS_PATH, SNAPSHOTS_PATH ]) {
     const git = new Git(repoPath);
-    await git.init();
+    await git.init(); /* eslint-disable-line no-await-in-loop */
   }
 }
 
 async function eraseRepo() {
   const promises = [];
 
   for (const repoPath of [ VERSIONS_PATH, SNAPSHOTS_PATH ]) {
-    const files = await fs.readdir(repoPath, { withFileTypes: true });
+    const files = await fs.readdir(repoPath, { withFileTypes: true }); /* eslint-disable-line no-await-in-loop */
 
-    promises.push(...files.map(file => {
+    promises.push(...files.map((file) => {
       const filePath = path.join(repoPath, file.name);
 
       if (file.isDirectory()) {
         return fs.rmdir(filePath, { recursive: true });
-      } else if (file.name !== 'README.md') {
+      } if (file.name !== 'README.md') {
```

Comment:
I don't understand why this has changed. The condition is not the same.

If the change is intentional, then shouldn't we at least have the `if` on its own line, separate from the closing curly bracket? 🤔

Reply:
It return the else because there is a return in the first `if`.
https://eslint.org/docs/rules/no-else-return